relay-general/src/store/normalize/spans.rs

```diff
@@ -0,0 +1,727 @@
+use std::collections::BTreeSet;
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+use crate::protocol::{Event, Span, Timestamp};
+use crate::types::Annotated;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub enum SpanAttribute {
+    #[serde(rename = "exclusive-time")]
```

Comment:
You can put a `serde(rename_all = "kebab-case")` onto the enum.

Please also add doc comments since this is exported across module boundaries.